arviz/plots/bpvplot.py

```diff
@@ -98,7 +98,7 @@ def plot_bpv(
         For example, ``data_pairs = {'y' : 'y_hat'}``
         If None, it will assume that the observed data and the posterior/prior
         predictive data have the same variable name.
-    labeller : labeller instance, optional
+    Labeller : Labeller instance, optional
```

Comment:
```suggestion
    labeller : Labeller, optional
```

The parameter name is using uppercase L which should be lowercase. The parmeter type uses uppercase which is correct but also has the `instance` part which is not.